recipes/bamsurgeon/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "Bamsurgeon" %}
+{% set version = "1.4.1" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/adamewing/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 137981f16df21fe5b4f602c641372a20643a8bb92e6c6dddace499427417213f
+
+build:
+  run_exports:
+    - {{ pin_subpackage('bamsurgeon', max_pin="x") }}
+  noarch: generic
+  number: 0
+
+requirements:
+  host:
+    - python >=3.0
+    - pip
+  run:
+    - python >=3.0
+    - samtools
+    - bwa
+    - exonerate >=2.4.0
+    - velvet
+    - pysam
+    - picard >=1.131
+
+test:
+  commands:
+    - ls
+    # test included in run_test.sh as find is not working and change of folder via cd not possibe in run_test.sh generated with this test here

```

Comment:
I don't understand. Why not test the three main command line tools (addindel, addsnv, addsv) provided by bamsurgeon directly?

Reply:
When I tested it I had the following problem mentioned also in an earlier comment: bamsurgeon is a standalone python-tool, so unlike neat no callable via the terminal. Therefore the correct folder must be chosen, and this was the problem. I used several commands, and also cloned the temporary virtual environment to directly run the test within my terminal.

When building a environment the folder for bamsurgeon gets a complicated name like "work_moved_bamsurgeon-1.4.1-py312hdfd78af_0_noarch". I suppose it might be possible to get the folders name via bash commands, pass them and navigate by this to the folder with the addsv.

All these within the test of the bamsurgeon meta.yaml caused problems with the cd command and didn't work. So I created a single run_test.sh file to use, which works.